sdk/communication/azure-communication-rooms/tests/test_rooms_client.py

```diff
@@ -90,9 +90,8 @@ def test_create_room_only_participants(self):
 
         self.verify_successful_room_response(response=response)
 
-    @pytest.mark.live_test_only
     @recorded_by_proxy
-    def test_create_room_validUntil_7Months_async(self):
+    def test_create_room_validUntil_7Months(self):
```

Comment:
could this be test_create_room_validUntil_seven_months or are numbers here fine?

Reply:
I can update this for consistency, thanks!

Reply:
I was using exceed_180days